pkg/controllers/clusterresourceplacement/controller_integration_test.go

```diff
@@ -0,0 +1,512 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+package clusterresourceplacement
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	placementv1beta1 "go.goms.io/fleet/apis/placement/v1beta1"
+	"go.goms.io/fleet/pkg/utils"
+)
+
+const (
+	member1Name = "member-1"
+	member2Name = "member-2"
+
+	eventuallyTimeout    = time.Minute * 2
```

Comment:
A nit: the eventually duration might be a little bit long.

Reply:
And same for the consistently duration as well.

Reply:
i feel having prefix is clearer. :)